### PR TITLE
Clean up code

### PR DESCRIPTION
**Pull request by AJenbo**
Type: refactoring  
Breaking change: no

### CODE REVIEW

File: src/main/php/PHPMD/Renderer/CheckStyleRenderer.php [hunk 1]
```diff
@@ -35,7 +35,7 @@ private function mapPriorityToSeverity($priority)
             return 'info';
         }
 
-        return (int) $priority === 2 ? 'warning' : 'error';
+        return $priority === 2 ? 'warning' : 'error';
```
Comment (kylekatarnls) [inline src/main/php/PHPMD/Renderer/CheckStyleRenderer.php:38]: I think we should do that only once `mapPriorityToSeverity(int $priority)` is updated to include the parameter type.

**Review by kylekatarnls — CHANGES_REQUESTED** [review]
https://github.com/phpmd/phpmd/pull/1184#discussion_r1632864905

**Review by kylekatarnls — APPROVED** [review]
Nice to have: https://github.com/phpmd/phpmd/pull/1184#discussion_r1632868964

File: src/main/php/PHPMD/RuleSetFactory.php [hunk 2]
```diff
@@ -339,8 +339,8 @@ private function parseSingleRuleNode(RuleSet $ruleSet, SimpleXMLElement $ruleNod
         if (isset($ruleNode['file'])) {
             if (is_readable((string) $ruleNode['file'])) {
                 $fileName = (string) $ruleNode['file'];
```
[the remark this answers is not in the document]
Comment (AJenbo) [reply src/main/php/PHPMD/RuleSetFactory.php:332]: Here you go: https://github.com/phpmd/phpmd/pull/1189

**Pull request merged**